ui/tests/e2e/test.js

```diff
@@ -124,7 +125,7 @@ describe("End-to-end", () => {
     expect(chip.length).toBe(1);
     // Wait for the facet card to be rendered and then assert.
     await waitForFacetCard("Avuncular");
-    await assertFacet("Avuncular", "10", "HG00658 (aunt/uncle)", "1");
+    await assertFacet("Avuncular", "46", "HG00658 (aunt/uncle)", "1");
```

Comment:
This shouldn't change (10 -> 46).

@bfcrampton , do you mind looking at the bug in the UI tests that I mentioned earlier? I wonder if that's related. I have a meeting now, I'll be in touch after the meeting.

Reply:
Sure looking into it now.

Reply:
Sorry I looked into this and 46 is right, since we're returning more facet values now. Please merge.